src/module-elasticsuite-indices/Model/Healthcheck/PrimaryShardsConfigCheck.php

```diff
@@ -69,7 +70,7 @@ public function __construct(
         UrlInterface $urlBuilder,
         int $sortOrder = 20
     ) {
-        parent::__construct($urlBuilder, $sortOrder);
+        parent::__construct($urlBuilder, $sortOrder, MessageInterface::SEVERITY_MINOR);
```

Comment:
Please add the severity is as an exposed constructor parameter, after the sortOrder.